docs/guides/ui/building_leader_board.md

```diff
@@ -18,7 +18,6 @@ Bookmark the leaderboard in the browser to reuse and share the leaderboard URL w
 For this tutorial, use one of the following as a project:
```

Comment:
@pollfly Is it "on of the following" now there's only one left?

Reply:
I see two - using your own project or executing example scripts (... to create your own project)